docs/monitoring/metrics.md

```diff
@@ -1370,6 +1370,147 @@ Thus, in order to infer the metric identifier:
   </tbody>
 </table>
 
+### System resources
+
+Logging of system resources is disabled by default. When `taskmanager.debug.additional-logging`
+is enabled on a TaskManager additional metrics listed below will be available. System
+resources metrics are updated periodically and they present average values for a configured
+interval (`taskmanager.debug.additional-logging-interval`).
+
+Logging of system resources require couple of optional dependencies to be present on the
+classpath (for example placed in Flink's `lib` directory):
+
+  - `com.github.oshi:oshi-core:3.4.0` (licensed under EPL 1.0 license)
+  - `net.java.dev.jna:jna-platform:jar:4.2.2`
```

Comment:
are these transitive dependency of `oshi-core`?

Reply:
yes. I have rephrased this paragraph.